cove/input/models.py

```diff
@@ -16,8 +18,16 @@
     'text/xml': 'xml',
 }
 
+def original_file_name(file_extension):
```

Comment:
I'd like to see the original file name still preserved in the string if possible. This improves ux because if you've done quite a number of these and have a downloads folder full of `quai3aenu8Xai5oh.xlsx` files it gets quite difficult to humanly identify them. Suggested change would be ```return f"{filename}-{random_stirng}.{file_extension}" ``` with filename being an argument to `originial_file_name`

Reply:
If a user uploads multiple files and downloads many corresponding files, it'll be very annoying to untease original-garbagetext1 from original-garbagetext2. It'd be a better user experience to just put a view in from of the /media/ files for which the URL is not guessable - while still sending back the filenames as before.